rlmolecule/sql/tables.py

```diff
@@ -28,7 +28,7 @@ class StateStore(Base):
 class GameStore(Base):
     __tablename__ = 'Game'
 
-    id = Column(String(32), primary_key=True)  # UUID
+    id = Column(String(64), primary_key=True)  # UUID
```

Comment:
I tested the QED example on eagle with yuma, and the ID column was too short. Changing it to 64 fixed the problem.

Reply:
Ah ok, odd. These should be uuid's, and postgres even has a UUID datatype. But I don't think sqlalchemy has a builtin way of handling the case where uuid types are not supported.
https://stackoverflow.com/questions/183042/how-can-i-use-uuids-in-sqlalchemy

Reply:
Ah OK, odd. We could probably even look into a UUID-specific datatype for that column: https://docs.sqlalchemy.org/en/13/core/custom_types.html#backend-agnostic-guid-type